a11y.js

```diff
@@ -36,6 +36,9 @@ const pageTypes = {
     "//div[@id='root']/header/nav/div/div[1]/div/ul",
   ],
   idxPage: ["//div[@id='root']/header/nav/div/div[1]/div/ul"],
+  onDemandTV: [
+    '/iframe', // Can be removed once https://github.com/bbc/bbc-a11y/issues/298 is resolved
```

Comment:
I believe it may have already been resolved - https://github.com/bbc/bbc-a11y/pull/313, so may not need to be included anymore. Check with Jonathan?

Reply:
@pharingee Removing this shows 33 errors instead of 15